Chummer/Classes/ColorManager.cs

```diff
@@ -224,6 +224,22 @@ private static Color TransformToDarkModeValidVersion(Color objColor)
         private static Color GrayHasNotesColorDark => GenerateDarkModeColor(GrayHasNotesColorLight);
         public static Color ErrorColor => Color.Red;
 
+        public static Color DieGlitchFore => IsLightMode ? DieGlitchForeLight : DieGlitchForeDark;
+        public static Color DieGlitchBackground => IsLightMode ? DieGlitchBackgroundLight : DieGlitchBackgroundDark;
+        public static Color DieHitFore => IsLightMode ? DieHitForeLight : DieHitForeDark;
+        public static Color DieHitBackground => IsLightMode ? DieHitBackgroundLight : DieHitBackgroundDark;
+
+        private static Color DieGlitchForeLight => Color.White;
+        private static Color DieGlitchBackgroundLight => Color.DarkRed;
+        private static Color DieHitForeLight => Color.Black;
+        private static Color DieHitBackgroundLight => Color.LightGreen;
```

Comment:
Bad choice of colors because of red-green colorblindness. There's a reason why the two colors used in the Karma/Nuyen charts are red and blue.

At the same time, does adding color information really help with legibility at a glance? The form already reports how many hits you have and whether or not you glitched. I don't see how coloring the background of individual 1, 5, and 6 entries (with 2's and 4's as well depending on the selected options) in the list is helpful TBH.

Reply:
I personally don't use the dice tool at all, but played around with it after yesterdays Issue that referenced it.
Some coloring would have been nice, because the list was the first thing I looked at after rolling some dice.

Reply:
I tried to apply the Red/Blue and DarkRed/Blue colors. Finally I now think that Gray/Green makes more sense, but I can change it once more if you like.
Regarding the question if it is useful: It at least allows to identify relevant parts more quickly.